src/core/utils/spec-preprocess.ts

```diff
@@ -524,11 +524,16 @@ export function overrideDataTemplates(spec: GoslingSpec) {
         switch (t.data.type) {
             case 'vector':
             case 'bigwig':
-                ts[i] = Object.assign(getVectorTemplate(t.data.column, t.data.value), t);
+                ts[i] = Object.assign(getVectorTemplate(t.data.column ?? 'position', t.data.value ?? 'value'), t);
```

Comment:
Ignore changes in this file. I will remove this `overrideDataTemplates()` function in a separate PR as we will make data templates deprecated. So, default values for `column`, `value`, and `row` are assigned in a single place, i.e., `data-abstraction.ts`.